var/spack/repos/builtin/packages/py-non-regression-test-tools/package.py

```diff
@@ -18,8 +18,8 @@ class PyNonRegressionTestTools(PythonPackage):
 
     version("develop", branch="develop")
     version("main", branch="main")
+    version("1.1.6", tag="v1.1.6")
```

Comment:
Is there a reason the `py-pysetuptools` version range is more recent than the one in `pyproject.toml`?

```
requires = ["setuptools >= 61.0"]
```

Reply:
Thanks. It is fixed in last push.